Author: Julien-externe BLACHE <julien-externe.blache@edf.fr>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=584098
Description: Fix to allow ESC key to work in whiptail.
Last-Updated: 2010-06-21
Forwarded: no

Index: newt-0.52.19/form.c
===================================================================
--- newt-0.52.19.orig/form.c
+++ newt-0.52.19/form.c
@@ -479,6 +479,7 @@ newtComponent newtForm(newtComponent ver
     if (!(form->flags & NEWT_FLAG_NOF12)) {
 	newtFormAddHotKey(co, NEWT_KEY_F12);
     }
+    newtFormAddHotKey (co, NEWT_KEY_ESCAPE);
 
     if (vertBar)
 	form->vertBar = vertBar;
